eslint.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = [
+  ...require('@railgun-reloaded/eslint-config')(),
   {
     ignores: [
       '**/src/generated/**/*.ts',
```

Comment:
I think this needs to stay, since there's generated code inside this directory which goes against some rules (unused variables mostly), and this is the best bang for the buck among possible solutions (ie: editing the file every time after generating code, etc)